v7/_posts/release notes/2018-01-11-ODL-7.4.0.md

```diff
@@ -11,22 +11,54 @@ category: "4. Release Notes"
 
 ODataLib 7.4.0 has the following beta releases:
 
-### ODataLib 7.4.0.beta & beta2
+### ODataLib 7.4.0.beta
 
 #### Features ##
 
-[[Issue #226](https://github.com/OData/odata.net/issues/226)] Support $batch in JSON format.
+[[Issue #103](https://github.com/OData/odata.net/issues/103)] OData v4: Deserialize client unknown properties into OData Object Model.
+
+[[Issue #801](https://github.com/OData/odata.net/issues/801)] Supporting Primitive Type Casts.
 
 [[Issue #988](https://github.com/OData/odata.net/issues/988)] Add support for 4.01 Delta Format
 
 #### Fixed Bugs ##
 
+[[Issue #698](https://github.com/OData/odata.net/issues/698)] DataServiceQuerySingle<T>.GetValueAsync inconsistent with GetValue in support for GET returning 404.
+
+[[Issue #800](https://github.com/OData/odata.net/issues/800)] Need for parsing Open types using OData.NET v7.X.
+
+[[Issue #949](https://github.com/OData/odata.net/issues/949)] ODataUriExtensions.BuildUri ignores $skiptoken.
+
+[[PR #961](https://github.com/OData/odata.net/pull/961)] Add support for ENUM keys.
+
+[[Issue #965](https://github.com/OData/odata.net/issues/965)] Parsing encoded character with special meaning in ODataJsonLightContextUriParser.
+
+
+### ODataLib 7.4.0.beta2
+
+#### Features ##
+
+[[Issue #226](https://github.com/OData/odata.net/issues/226)] Support a json serialization for $batch.
+
+[[Issue #866](https://github.com/OData/odata.net/issues/866)] "Microsoft.OData.Client" support for dot net core.
```

Comment:
Maybe better as " .NET Core" ? 

And do we specific add the support Core version?